O/osslsigncode/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "osslsigncode"
+version = v"2.9.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mtrojnar/osslsigncode",
+                  "76ee550c9d3b9f0e559f044e18136b74c167fef2"),
+]
+
+script = raw"""
+    cd $WORKSPACE/srcdir/osslsigncode
+
+    sed -i 's/Ws2_32/ws2_32/g' CMakeLists.txt
+
+    cmake -B build \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DZLIB_INCLUDE_DIR=${includedir} \
+        -DZLIB_LIBRARY=/workspace/destdir/lib/libz.${dlext}
```

Comment:
Please ***always*** use the most specific [environment variable](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables):
```suggestion
        -DZLIB_LIBRARY="${libdir}/libz.${dlext}"
```
Don't use absolute paths, absolute paths are a bad idea.

Reply:
Ok, I will take a note.